app/models/post.rb

```diff
@@ -0,0 +1,41 @@
+class Post < ApplicationRecord
+  belongs_to :user
+
+  # Not Null制約を付けたら必須のバリデーションはつけるべきと考えたので付けた。
+  validates :images, presence: true
+  validates :body, presence: true, length: { maximum: 100 }
+
+  # 複数枚画像を投稿する時は、mount_uplodersと言う風にsを付けて複数形にする。こうする事で、配列で格納できる様になる。
+  # 最初sを飛ばしていて、nullに対してeachメソッドを使う形になっていたのでundifind_methodがでた。sを付けたら空配列に対してeach文なので出ない。
+  mount_uploaders :images, PostImageUploader
+  # serializeを付けずに投稿するとURLとして認識されてない。
+  serialize :images, JSON
+  # 並び替えは結構使いそうだから、とりあえずscopeで切り出した。
+  scope :recent, -> { order(created_at: :desc) }
+
+  # 遊びメソッド。ダミーusername生成用。名前漏れてたらすみません
+  def self.dummy_name_for_sparta
```

Comment:
```suggestion
  def self.dummy_name_for_sparta(count)
```
みたいに呼び出し側から人数を制御できた方が便利じゃないですかね？

Reply:
確かに！見本が4人表示だったんで決めうちしちゃいました笑
どうせなら便利になる様に仕様変更してみます！